test/runtests.jl

```diff
@@ -1990,27 +1991,45 @@ end
     lm1 = lm(@formula(Prestige ~ 1 + Income + Education), duncan)
     @test termnames(lm1)[2] == coefnames(lm1)
     @test vif(lm1) ≈ gvif(lm1)
-    
+
     lm1_noform = lm(modelmatrix(lm1), response(lm1))
     @test vif(lm1) ≈ vif(lm1_noform)
     @test_throws ArgumentError("model was fitted without a formula") gvif(lm1_noform)
-    
+
     lm1log = lm(@formula(Prestige ~ 1 + exp(log(Income)) + exp(log(Education))), duncan)
     @test termnames(lm1log)[2] == coefnames(lm1log) == ["(Intercept)", "exp(log(Income))", "exp(log(Education))"]
     @test vif(lm1) ≈ vif(lm1log)
-    
+
     gm1 = glm(modelmatrix(lm1), response(lm1), Normal())
     @test vif(lm1) ≈ vif(gm1)
-    
+
     lm2 = lm(@formula(Prestige ~ 1 + Income + Education + Type), duncan)
     @test termnames(lm2)[2] != coefnames(lm2)
     @test gvif(lm2; scale=true) ≈ [1.486330, 2.301648, 1.502666] atol=1e-4
-    
+
     gm2 = glm(@formula(Prestige ~ 1 + Income + Education + Type), duncan, Normal())
     @test termnames(gm2)[2] != coefnames(gm2)
-    @test gvif(gm2; scale=true) ≈ [1.486330, 2.301648, 1.502666] atol=1e-4   
-    
+    @test gvif(gm2; scale=true) ≈ [1.486330, 2.301648, 1.502666] atol=1e-4
+
     # the VIF definition depends on modelmatrix, vcov and stderror returning valid
     # values. It doesn't care about links, offsets, etc. as long as the model matrix,
     # vcov matrix and stderrors are well defined.
 end
+
+@testset "NIST - Filip. Issue 558" begin
+    # Since "https://www.itl.nist.gov/div898/strd/lls/data/LINKS/DATA/Filip.dat" seems to block download, we'll use a "mirror"
+    fn = Downloads.download("https://gist.githubusercontent.com/andreasnoack/9dadfb922fc45ca52bc7b2659b7c5b67/raw/f1dcdd2179b606ecbb7e23316da1ede971e7ffe8/Filip.dat")
```

Comment:
There's https://github.com/joshday/NISTStatisticalReferenceData.jl/ which however seems to use `Downloads.download` for downloading the datasets (and storing them in `deps/` seems a bit suboptimal).

Reply:
Maybe there will be some better way of doing this in the future. Then we can adjust.